x/cheqd/types/tx_msg_update_did_test.go

```diff
@@ -50,13 +68,15 @@ func TestMsgUpdateDidValidation(t *testing.T) {
 					VersionId:      "version1",
 				},
 				Signatures: nil,
-			},
-			isValid:  false,
-			errorMsg: "payload: (authentication: there should be no duplicates.).: basic validation failed",
-		},
-		{
-			name: "negative: version id is required",
-			struct_: &MsgUpdateDid{
+			}
+			isValid = false
+			errorMsg = "payload: (authentication: there should be no duplicates.).: basic validation failed"
+		})
+	})
+
+	When("VersionId is empty", func() {
```

Comment:
Isn't VersionID set by ledger?

Reply:
We have to set VersionID for avoiding race condition and for point which particular "snapshot" of DID we want to edit/update